packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -1720,13 +1720,33 @@ class Flexible extends ParentDataWidget<Flex> {
 
 /// A paragraph of rich text.
 ///
-/// Consider using [Text], which integrates with [DefaultTextStyle], rather than
-/// this widget, which requires explicit styling.
+/// The [RichText] widget displays text using a variety of different styles. The
```

Comment:
not sure "using a variety of different styles" quite conveys the right impression here.